grails-app/views/layouts/_footer.gsp

```diff
@@ -37,7 +37,7 @@
 <asset:javascript src="jquery.fn.dataTablesExt.ticket.js"/>
 <asset:javascript src="chosen.jquery.min.js"/>
 <asset:javascript src="jasny-bootstrap.min.js"/>
-<script src="https://cloud.tinymce.com/5/tinymce.min.js?apiKey=8zknubfnjvv9l3sg0cpxrome1qk6r2wlpdw7j4ebb3gjxige"></script>
+<script src="https://cloud.tinymce.com/5/tinymce.min.js?apiKey=kyknyn3xmeam53u8vyway2oxf18oecyfwkjoym1xcydpyfyh"></script>
```

Comment:
Is this also necessary for the version deployed to the Broad VM or is it only for the cloud version? Also, is it necessary at all? I thought that Leo submitted a PR to replace this.

Reply:
Answering my own question, we're using an updated version of tinymce (https://github.com/tinymce/tinymce-react) so we do still need an API key with this PR: https://github.com/broadinstitute/orsp-pub/pull/208